### PR TITLE
Query module using several deprecated configuration methods from Infinispan core

### PR DESCRIPTION
**Pull request by Sanne**
https://issues.jboss.org/browse/ISPN-8035

### CODE REVIEW

**Comment by Sanne** [comment]
rebased

File: query/src/test/java/org/infinispan/query/directoryinteraction/IndexStoredIndexedCacheTest.java [hunk 1]
```diff
@@ -48,7 +48,6 @@ protected void createCacheManagers() throws Throwable {
       .invocationBatching()
          .disable()
       .deadlockDetection()
```
Comment (anistor) [inline query/src/test/java/org/infinispan/query/directoryinteraction/IndexStoredIndexedCacheTest.java]: ```deadlockDetection()``` should be removed completely.

**Comment by anistor** [comment]
LGTM.

File: query/src/test/java/org/infinispan/query/directoryinteraction/IndexStoredIndexedCacheTest.java (hunk 1 above)
Comment (Sanne) [inline query/src/test/java/org/infinispan/query/directoryinteraction/IndexStoredIndexedCacheTest.java]: ok.. but it's not deprecated. Need a JIRA for Infinispan core?

Comment (Sanne) [inline query/src/test/java/org/infinispan/query/directoryinteraction/IndexStoredIndexedCacheTest.java]: ah, IC what you mean now.

Comment (anistor) [inline query/src/test/java/org/infinispan/query/directoryinteraction/IndexStoredIndexedCacheTest.java]: this: https://github.com/infinispan/infinispan/blob/master/core/src/main/java/org/infinispan/configuration/cache/ConfigurationChildBuilder.java#L21

**Comment by anistor** [comment]
I fixed it and merged it into master. Thanks @Sanne !

**Pull request closed without merging**